guides/source/active_storage_overview.md

```diff
@@ -876,8 +876,8 @@ specific:
 ```
 
 [`variant`]: https://api.rubyonrails.org/classes/ActiveStorage/Blob/Representable.html#method-i-variant
-[`web images`]: https://github.com/rails/rails/blob/main/activestorage/lib/active_storage/engine.rb#L47
-[`variable`]: https://github.com/rails/rails/blob/main/activestorage/lib/active_storage/engine.rb#L34
+[`web images`]: https://github.com/rails/rails/blob/af7428c4acd0fcf9eed5c7215856594a078f23b7/activestorage/lib/active_storage/engine.rb#L47
```

Comment:
Thanks for fixing this.
Too bad we can't deeplink to the exact configuration: https://edgeguides.rubyonrails.org/configuring.html#configuring-active-storage

Reply:
I will try to add an anchor to that section.  I am also investigating if it is possible to extract those arrays to constants and link them on the RDOC.

Reply:
Converting the links to headers will allow link them. In any case, this config can be outdated, so I want to investigate if it is possible to link constants inside the RDOC documentation. 

We could add a comment in the source code to remember to update the guides, but this strategy would pollute the codebase, so not sure If I like it.

Reply:
@ceritium I like that idea!  Converting those list items to sub-headings could also help prevent formatting mistakes like in the screenshots in #40921.  (And it might also be possible to fix the ugly wrapping in https://github.com/rails/rails/pull/40921#discussion_r548268402.)

In any case, I think explicitly naming the config values would be helpful:

```diff
index 9304b94d7e..b58afce6c2 100644
--- a/guides/source/active_storage_overview.md
+++ b/guides/source/active_storage_overview.md
@@ -853,7 +853,9 @@ location.
 If a variant is requested, Active Storage will automatically apply
 transformations depending on the image's format:
 
-1. Content types that are [`variable`] and not considered [`web images`], will be converted to PNG.
+1. Content types that are variable (as dictated by `config.active_storage.variable_content_types`)
+    and not considered web images (as dictated by `config.active_storage.web_image_content_types`)
+    will be converted to PNG.
 2. If `quality` is not specified, the variant processor's default quality for the format will be used.
 
 The default processor for Active Storage is MiniMagick, but you can also use
```

With the above change, "`config.active_storage.variable_content_types`" and "`config.active_storage.web_image_content_types`" could link to their respective sub-headings, or they could link to the [Configuring Active Storage](https://guides.rubyonrails.org/configuring.html#configuring-active-storage) section, or they could stand alone and not link to anything.

Reply:
I am going to replace the weak links with the config names as you suggested.

Reply:
Since I removed the weak links, I will mark this conversation as resolved, I had no luck with the "RDOC" (it takes too long), and I prefer not to change from lists to headers if it is not necessary. Thanks!